### ISSUE TITLE
Addition of EFSOI block to anavinfo

### ISSUE THREAD

**Issue by AndrewEichmann-NOAA** [NOAA-EMC/GSI#444]
When the EFSOI task runs the executable reads in anavinfo and expects the following block:

```
state_vector_efsoi::
!var     level  itracer source     funcof
 u       127      0     met_guess    u
 v       127      0     met_guess    v
 tv      127      0     met_guess    tv
 q       127      1     met_guess    q
 ps        1      0     met_guess    prse
::
```
Presently a version of global_anavinfo.l127.txt  is kept in my fork of GSI-utils, but at some point this should be part of the regular fix files. What needs to be done to get this into at least global_anavinfo.l127.txt (and ideally probably all of the global anavinfo files)?

**Comment by MichaelLueken** [comment]
@AndrewEichmann-NOAA To add or update fix files for the GSI, please see:

https://vlab.noaa.gov/redmine/projects/comgsi/wiki/Updating_fix_files_in_ProdGSI

After cloning your GSI fork:

- cd fix
- git checkout rev2
- git checkout -b feature/efsoi_anavinfo

Copy your updated `global_anavinfo.l127.txt` file to the new fix branch and apply modifications to the rest of the anavinfo files (if desired at this time).

- git add global_anavinfo.l127.txt
- git push origin feature/efsoi_anavinfo

I will then be able to merge the `feature/efsoi_anavinfo` fix branch to `rev2` once the modifications have been made and pushed and there is no work out to the review committee.

**Comment by AndrewEichmann-NOAA** [comment]
@MichaelLueken-NOAA Thank you, done.

**Comment by MichaelLueken** [comment]
@AndrewEichmann-NOAA I see `feature/efsoi_anavinfo` in the fix submodule, but it looks like the `global_anavinfo.l127.txt` file hasn't been updated (there is no `state_vector_efsoi` entry and there is no new entry in the commit log).  Please use git commit and then push back to `feature/efsoi_anavinfo` branch one more time.  I will then be able to merge `feature/efsoi_anavinfo` into `rev2` tomorrow after the current work out to review has been merged to the authoritative `develop` branch.

**Comment by AndrewEichmann-NOAA** [comment]
@MichaelLueken-NOAA Sorry, forgot to commit it. I did so and pushed it.

**Comment by MichaelLueken** [comment]
The `feature/efsoi_anavinfo` branch was merged to `rev2` at `023f81d` of the fix submodule.  This was pushed to the authoritative `develop` branch at [e23204e](https://github.com/NOAA-EMC/GSI/commit/e23204e5ea8f73056ffb2d181ff340deaa0d2163).  I will now close this issue.

**Comment by AndrewEichmann-NOAA** [comment]
Thank you!
